forks/jdt/pom.xml

```diff
@@ -19,7 +19,7 @@
             <groupId>org.eclipse.tycho</groupId>
             <artifactId>org.eclipse.jdt.core</artifactId>
             <version>3.10.0.v20140604-1726</version>
-            <scope>provided</scope>
+<!--             <scope>provided</scope> -->
```

Comment:
This will have the side effect of pulling this into the windup distribution again, creating a conflict between our modified version and the upstream version.

I don't think we can leave it this way for the merge, although maybe it is your intention to remove it before merging?